terraform/eks/main.tf

```diff
@@ -250,6 +250,7 @@ module "validator" {
     module.aoc_oltp,
     module.adot_operator,
     kubectl_manifest.logs_sample_fargate_deploy,
-    null_resource.prom_base_ready_check
+    null_resource.prom_base_ready_check,
+    kubernetes_deployment.aoc_deployment
```

Comment:
LGTM, Is this tested in local?

Reply:
No, I will monitor the collector CI though and adjust if necessary.